### PR TITLE
SplitButton: High contrast border fix

### PR DESCRIPTION
**Pull request by maxwellred**
#### Pull request checklist

- [x] Addresses an existing issue: #10299 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Copies over solution from #10364 

#### BEFORE
image

#### AFTER
Split button high contrast (1)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10656)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts [hunk 1]
```diff
@@ -70,6 +70,10 @@ export function standardStyles(theme: ITheme): IButtonStyles {
               color: 'Highlight'
             }
           }
+        },
+        [HighContrastSelector]: {
+          border: `1px solid white`,
+          borderLeft: 'none'
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts]: Why is this here? This is different from the styling in #10364.

**Comment by khmakoto** [comment]
@maxwellred Can you run `yarn update-snapshot` on the `apps/a11y-tests` folder? It seems the build is complaining about snapshots being changed.

**Comment by aneeshack4** [comment]
@maxwellred Pinging since there hasn't been activity for a while now

File: packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts (hunk 1 above)
Comment (aneeshack4) [reply packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts]: Hmm yeah I don't see this in the original fabric 7 change https://github.com/OfficeDev/office-ui-fabric-react/pull/10364/files#diff-83f447f02b4959069f9feb281f1c83b2R98

Comment (maxwellred) [reply packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts]: In Fabric 7 a border is introduced to the button in non-HC mode. Since it's not there in Fab 6 i have to add this white border only in HC for there to be one.

**Comment by khmakoto** [comment]
@maxwellred once the CI passes I'll check the examples locally and, if everything looks good, I'll approve. Thanks!

File: packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts (hunk 1 above)
Comment (aneeshack4) [reply packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts]: Slightly confused - why does this need to be added in Fabric 6 if the border is only expected for Fabric 7? There are screener changes down below so we should be careful not to break anyone still using Fabric 6.

Comment (khmakoto) [reply packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts]: I think this is because in Fabric 6 we don't use a border on the `Button` due to the background color indicating where it ends. However, this background color is no longer visible in High Contrast Mode, which is why we need a border there.
